test-case/run-all-tests.sh

```diff
@@ -39,15 +35,27 @@ suspend-resume-with-capture"
 main()
 {
 	local failures=0
+	local time_delay=3
+	local tplg_name
+
+	while getopts "ht:T:" OPTION; do
+	case "$OPTION" in
+		t) tplg_name="$OPTARG" ;;
+		T) time_delay="$OPTARG" ;;
+		*) usage; exit 1 ;;
+		esac
+	done
+	if [ -z "$TPLG" ]; then
+		export TPLG="/lib/firmware/intel/sof-tplg/$tplg_name"
+	fi
```

Comment:
I think this gives the environment precedence over the (more specific) command line parameter, so it's not OK.

Do we really need this `-t` option? I mean do we really need to do anything with TPLG in this file? I think all other tests use the $TPLG environment variable, correct? Why not just do nothing here and stay consistent with all other tests?

BTW why not pass-through "$@" to every test?

Reply:
OK, I can remove the -t option, and just use TPLG to pass through topology path to all test case. We can not use  "$@" here, because every single test case takes different parameters.

Reply:
> We can not use "$@" here, because every single test case takes different parameters.

I've seen some options that _seem_ to be present in every test (`git grep OPT_`). I admittedly haven't checked whether some options actually exist in every single test.